src/www/index.html

```diff
@@ -178,6 +174,21 @@
       }
     }
   </style>
+  <script>
+    var browserSuggestionTweak;
+    $(document).on('mouseover', function () {
+        /* Clearing any timeout set under browserSuggestionTweak */
+        clearTimeout(browserSuggestionTweak);
+        $('.overlay')[0].style.opacity = 1;
+    });
+    $(document).on('mouseout', function () {
+        /* Set a timeout as immediatly displaying the overlay when suggestions
+        appear in the browser will be very bad.*/
+        browserSuggestionTweak = setTimeout(function () {
+            $('.overlay')[0].style.opacity = "";
+        }, 3000);
```

Comment:
Lets not add timeout. Let it be instantaneous.

Reply:
Making it instantaneous proovides this bug @geekyd. If you could pls see the issue #1591